applications/RomApplication/python_scripts/petrov_galerkin_training_utility.py

```diff
@@ -0,0 +1,158 @@
+# Import Python modules
+import json
+import numpy as np
+import os
+import scipy.linalg
```

Comment:
```suggestion
```
I think it is not used. Besides, in case of requiring `scipy` you should check the availability of the module.